cmake/BLTMacros.cmake

```diff
@@ -272,7 +343,22 @@ macro(blt_add_library)
         set(arg_CLEAR_PREFIX TRUE)
     endif()
 
-    if ( arg_SHARED OR ENABLE_SHARED_LIBS )
+
+    #
+    #  cuda support
+    #
+    list(FIND arg_DEPENDS_ON "cuda" check_for_cuda)
```

Comment:
Thanks I over looked that, but I gave it a spin (with CMake 3.4):

```
CMake Warning (dev) at blt/cmake/BLTMacros.cmake:350 (if):
  Policy CMP0057 is not set: Support new IN_LIST if() operator.  Run "cmake
  --help-policy CMP0057" for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

  IN_LIST will be interpreted as an operator when the policy is set to NEW.
  Since the policy is not set the OLD behavior will be used.
Call Stack (most recent call first):
  blt/thirdparty_builtin/fruit-3.3.9/CMakeLists.txt:9 (blt_add_library)
This warning is for project developers.  Use -Wno-dev to suppress it.

CMake Error at blt/cmake/BLTMacros.cmake:350 (if):
```

I agree that would be ideal, but given that we have to set the policy to make it work, I don't want to mess with it.